C/common/config_category.cpp

```diff
@@ -138,9 +138,11 @@ ConfigCategory::ConfigCategory(const string& name, const string& json) : m_name(
 {
 	Document doc;
 	doc.Parse(json.c_str());
+	// REMOVE ME
+	Logger::getLogger()->error("ConfigCategory::ConfigCategory %s", json.c_str());
```

Comment:
```
"DefaultAFLocation" : {
 
  "type" : "string", 
  "value" : "/fledge\s/data_piwebapi/default",
  "default" : "/fledge/data_piwebapi/default", 
```

Reply:
check C/services/common/management_api.cpp which received the change callback payload.

Reply:
"value" : "/fledge\s/data_piwebapi/default"

this is wrong \s is not valid, it should be \\s

You can easily verify it
# echo '{"value" : "/fledge\s/data_piwebapi/default"}' | jq
parse error: Invalid escape at line 1, column 44

The string passed to configchange  must contain \\s


You can easily verify it

echo '{"value" : "/fledge\\s/data_piwebapi/default"}' | jq
{
  "value": "/fledge\\s/data_piwebapi/default"
}

Reply:
@pintomax Please see other details.